vite.config.js

```diff
@@ -1,11 +1,13 @@
 import { searchForWorkspaceRoot } from 'vite';
 import fs from 'fs';
+import react from '@vitejs/plugin-react';
 
 export default {
 
 	root: './example/',
 	base: '',
 	build: {
+		minify: false,
```

Comment:
Why are we disabling minification?

Reply:
Sorry this was to troubleshoot the lema controls error, caused by undefined value for access-keys